### PR TITLE
change the driver to sock mode

### PR DESCRIPTION
**Pull request by jikui**

### CODE REVIEW

File: bigip/create_bigip.sh [hunk 1]
```diff
@@ -194,6 +206,7 @@ write_files:
 
 runcmd:
   - /tmp/setup.sh >>/tmp/setup.log 2>&1 &
+  - /tmp/change_driver.sh >>/tmp/setup.log 2>&1 &
```
Comment (zhaoqin-github) [inline bigip/create_bigip.sh:209]: I think these two commands are executed at the same time. You may not garantee setup.sh can complete in 60s.

Comment (zhaoqin-github) [reply bigip/create_bigip.sh:209]: Maybe you can look at setup.done, or you may have other better choice...

Comment (jikui) [reply bigip/create_bigip.sh:213]: resolved

File: bigip/create_bigip.sh [hunk 2]
```diff
@@ -175,8 +175,12 @@ write_files:
       printf "\n" >> /root/.bash_profile
       cat /root/bash_profile_addon >> /root/.bash_profile
       rm -f /root/bash_profile_addon
-
       touch /tmp/setup.done
+      vendor_id=\`lspci -nn | grep -i eth | head -1 | cut -d' ' -f 11\` 
+      vendor_id=\`echo \${vendor_id#*[}\`
+      vendor_id=\`echo \${vendor_id%]*}\`
+      echo "device driver vendor_dev \$vendor_id sock" >> /config/tmm_init.tcl 
+      reboot
```
Comment (zhaoqin-github) [inline bigip/create_bigip.sh:183]: I am not quite sure about the difference of "reboot" and "tmsh reboot". Which one is the "official" command to restart the device?

File: bigip/create_bigip.sh [hunk 3]
```diff
@@ -175,8 +175,12 @@ write_files:
       printf "\n" >> /root/.bash_profile
       cat /root/bash_profile_addon >> /root/.bash_profile
       rm -f /root/bash_profile_addon
-
       touch /tmp/setup.done
```
Comment (zhaoqin-github) [inline bigip/create_bigip.sh:178]: Maybe we can move "touch /tmp/setup.done" to the line before reboot. Is it better?

File: bigip/create_bigip.sh (hunk 2 above)
Comment (zhaoqin-github) [inline bigip/create_bigip.sh:183]: Suggest to add a note here, which can indicate the reason why we have to restart the device. I guess maybe some of us will forget it next year.

Comment (jikui) [reply bigip/create_bigip.sh]: I verified that both "reboot" and "tmsh reboot" are ok. It seems "tmsh reboot" is better.

File: bigip/create_bigip.sh (hunk 3 above)
Comment (jikui) [reply bigip/create_bigip.sh]: yes, accepted it.

**Pull request merged**